pom.xml

```diff
@@ -58,9 +58,9 @@
         <bnd.version>5.1.2</bnd.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <aem.sdk.api>2022.5.7575.20220530T152407Z-220401</aem.sdk.api>
+        <aem.sdk.api>2022.9.8722.20220912T101352Z-220800</aem.sdk.api>
```

Comment:
This will need to be bumped again when the SDK that includes Core Components 2.21.0 is released.